veggieseasons/lib/screens/search.dart

```diff
@@ -70,17 +60,24 @@ class _SearchScreenState extends State<SearchScreen> {
           decoration: BoxDecoration(
             color: Styles.scaffoldBackground,
           ),
-          child: SafeArea(
-            child: Column(
-              children: [
-                _createSearchBox(),
-                Expanded(
-                  child: ListView(
-                    children: _generateVeggieRows(model.searchVeggies(_terms)),
-                  ),
+          child: Stack(
+            children: [
+              Positioned.fill(
+                child: ListView(
+                  padding: EdgeInsets.only(top: 100),
```

Comment:
Is there any more elegant way to do this beyond a hardcoded number?

Reply:
Ha, you totally caught me. Unfortunately there isn't a way to do it in the framework presently.
https://github.com/flutter/flutter/issues/12912

Reply:
That is to have one sibling's build depend on another sibling building and laying out first.